onnxruntime/core/providers/js/operators/unsqueeze.h

```diff
@@ -26,8 +26,9 @@ class Unsqueeze final : public JsKernel, public UnsqueezeBase {
     if (num_inputs == 2) {  // axes is an input
       const Tensor* axes_tensor = context->Input<Tensor>(1);
       ORT_ENFORCE(axes_tensor != nullptr, "Axes input is null");
-      ORT_ENFORCE(axes_tensor->Shape().NumDimensions() == 1,
-                  "An axes tensor must be a vector tensor.");
+      ORT_ENFORCE(axes_tensor->Shape().NumDimensions() == 0 ||
+                      axes_tensor->Shape().NumDimensions() == 1,
+                  "An axes tensor must be a scalar or a 1-D tensor.");
```

Comment:
nit: Maybe we should use the term vector instead of 1-D to be consistent with other places. The CPU unsqueeze is the only place to use 1-D (even CPU squeeze uses vector), and it should be fixed.

Reply:
The whole codebase is mixed with both "1-D tensor"(287 results) and "vector tensor"(23 results) in all different places. I would not do a cleanup for the whole codebase this time.

Reply:
I only checked the folder onnxruntime/core/providers. There are only 4 "1-D tensor", while the rest (14 hits) are "vector tensor", including unsqueeze js version. 
I don't mean you have to fix the CPU part, but maybe we can unify the js part.

Reply:
updated in the code